Makefile

```diff
@@ -20,6 +20,9 @@ docker-run:
 docker-run-with-druid:
 	docker compose -f docker-compose.yml -f docker-compose-druid.yml up
 
+docker-run-with-cockroachdb:
+	docker compose -f docker-compose.yml -f docker-compose.cockroachdb.yml up
```

Comment:
nit: either make it `docker-compose-cockroachdb.yml` or `docker-compose.druid.yml`

Reply:
Fixed, good catch 👍